sass/px-dropdown.scss

```diff
@@ -29,21 +25,21 @@ $inuit-enable-btn--disabled: true;
 $inuit-enable-btn--bare: true;
 $inuit-enable-btn--bare--primary: true;
 @import "px-buttons-design/_objects.buttons.scss";
+@import "px-shadows-design/_objects.shadows.scss";
 
 // Component
 :host {
   display: flex;
   position: relative;
-  text-align: left;
-  -webkit-tap-highlight-color: rgba(0,0,0,0);
+  // -webkit-tap-highlight-color: rgba(0,0,0,0);// <-- Why is this defined twice
```

Comment:
This appears to be an outdated way to achieve this. I think we can remove both lines because we have `outline: none` on the dropdown-trigger.